### PR TITLE
PHP 5.4 minimum. Add php-cs-fixer config and test.

### PR DESCRIPTION
**Pull request by oschwald**

### CODE REVIEW

File: src/Database/Reader.php [hunk 1]
```diff
@@ -57,14 +57,14 @@ public function __construct(
     /**
      * This method returns a GeoIP2 City model.
      *
-     * @param string $ipAddress IPv4 or IPv6 address as a string.
+     * @param string $ipAddress iPv4 or IPv6 address as a string
```
Comment (BassemN) [inline src/Database/Reader.php]: `iPv4` should be "IPv4"

File: src/Database/Reader.php [hunk 2]
```diff
@@ -74,14 +74,14 @@ public function city($ipAddress)
     /**
      * This method returns a GeoIP2 Country model.
      *
-     * @param string $ipAddress IPv4 or IPv6 address as a string.
-     *
-     * @return \GeoIp2\Model\Country
+     * @param string $ipAddress iPv4 or IPv6 address as a string
```
Comment (BassemN) [inline src/Database/Reader.php]: `iPv4` should be "IPv4"

File: src/Database/Reader.php [hunk 3]
```diff
@@ -91,14 +91,14 @@ public function country($ipAddress)
     /**
      * This method returns a GeoIP2 Anonymous IP model.
      *
-     * @param string $ipAddress IPv4 or IPv6 address as a string.
-     *
-     * @return \GeoIp2\Model\AnonymousIp
+     * @param string $ipAddress iPv4 or IPv6 address as a string
```
Comment (BassemN) [inline src/Database/Reader.php]: `iPv4` should be "IPv4"

File: src/Database/Reader.php [hunk 4]
```diff
@@ -112,14 +112,14 @@ public function anonymousIp($ipAddress)
     /**
      * This method returns a GeoLite2 ASN model.
      *
-     * @param string $ipAddress IPv4 or IPv6 address as a string.
+     * @param string $ipAddress iPv4 or IPv6 address as a string
```
Comment (BassemN) [inline src/Database/Reader.php]: `iPv4` should be "IPv4"

File: src/Database/Reader.php [hunk 5]
```diff
@@ -133,14 +133,14 @@ public function asn($ipAddress)
     /**
      * This method returns a GeoIP2 Connection Type model.
      *
-     * @param string $ipAddress IPv4 or IPv6 address as a string.
+     * @param string $ipAddress iPv4 or IPv6 address as a string
```
Comment (BassemN) [inline src/Database/Reader.php]: `iPv4` should be "IPv4"

File: src/Database/Reader.php [hunk 6]
```diff
@@ -154,14 +154,14 @@ public function connectionType($ipAddress)
     /**
      * This method returns a GeoIP2 Domain model.
      *
-     * @param string $ipAddress IPv4 or IPv6 address as a string.
+     * @param string $ipAddress iPv4 or IPv6 address as a string
```
Comment (BassemN) [inline src/Database/Reader.php]: `iPv4` should be "IPv4"

File: src/Database/Reader.php [hunk 7]
```diff
@@ -175,14 +175,14 @@ public function domain($ipAddress)
     /**
      * This method returns a GeoIP2 Enterprise model.
      *
-     * @param string $ipAddress IPv4 or IPv6 address as a string.
-     *
-     * @return \GeoIp2\Model\Enterprise
+     * @param string $ipAddress iPv4 or IPv6 address as a string
```
Comment (BassemN) [inline src/Database/Reader.php]: `iPv4` should be "IPv4"

File: src/Database/Reader.php [hunk 8]
```diff
@@ -192,14 +192,14 @@ public function enterprise($ipAddress)
     /**
      * This method returns a GeoIP2 ISP model.
      *
-     * @param string $ipAddress IPv4 or IPv6 address as a string.
-     *
-     * @return \GeoIp2\Model\Isp
+     * @param string $ipAddress iPv4 or IPv6 address as a string
```
Comment (BassemN) [inline src/Database/Reader.php]: `iPv4` should be "IPv4"

File: src/Database/Reader.php (hunk 1 above)
Comment (oschwald) [inline src/Database/Reader.php]: Thanks for noticing these!

**Pull request merged**